collections/_posts/2022-07-01-UNETR-TransformerMedicalImageSegmentation.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: review
+title: "UNETR: Transformers for 3D Medical Image Segmentation"
+tags: deep-learning CNN transformer segmentation medical
+author: "Pierre Rougé"
+cite:
+    authors: "Ali Hatamizadeh, Yucheng Tang, Vishwesh Nath, Dong Yang, Andriy Myronenko, Bennet Landman, Holger R. Roth, Daguang Xu"
+    title:   "UNETR: Transformers for 3D Medical Image Segmentation"
+    venue:   " IEEE Winter Conference on Applications of Computer Vision (WACV) 2022"
+pdf: "https://arxiv.org/abs/2103.10504"
+---
+
+# Notes
+
+* Code is available on [github](https://github.com/Project-MONAI/research-contributions/tree/main/UNETR/BTCV) and [monai](https://docs.monai.io/en/stable/networks.html#unetr)
+
+# Highlights
+
+* The goal is to take advantage of the transformer's capacity to learn long-range dependencies to overcome the limitations of CNNs 
+* The proposed architecture, UNETR (UNEt TRansfomers) uses a transfomer as an encoder and convolutional layer in the decoder to compute the segmentation output
+* The method is validated on the Multi Atlas Labeling Beyond The Cranial Vault (BTCV) dataset for multiorgan segmentation and the Medical Segmentation Decathlon (MSD) dataset for brain tumor and spleen segmentation tasks.
+* Method shows new state-of-the-art performance on the BTCV leaderboard.
+
+# Methods
+
+![](/collections/images/unetr/overview_method.jpg)
+
+## Architecture
+
+* The architecture is transformer based, so you can refer to this [tutorial](https://creatis-myriad.github.io./tutorials/2022-06-20-tutorial_transformer.html) if you need more details. 
+
+* 3D input volume $$\mathbf{x} \in \mathbb{R}^{H \times W \times D \times C}$$ is divided into non-overlapping patches of size $$(P, P, P)$$ which are flattened to give $$N$$ tokens arrange in a matrix $$\mathbf{x_v} \in \mathbb{R}^{N \times (P^3.C)}$$
+
+* A linear layer is used to project the patches into a $$K$$ dimensional embedding space, then a 1D learnable positional embedding is added giving $$\mathbf{z}_{0} \in \mathbb{R}^{N \times K}$$ :
+
+ $$ \mathbf{z}_{0} = [\mathbf{x}_{v}^{1}\mathbf{E}; \mathbf{x}_{v}^{2}\mathbf{E};...;\mathbf{x}_{v}^{N}\mathbf{E}] + \mathbf{E}_{pos}$$
+
+with 
+	 $$\mathbf{E} \in \mathbb{R}^{(P^3.C) \times K}$$ the projection matrix to learn
+
+* Note : No class token here because is not classification
```

Comment:
Note: no class token is involved in this architecture since the targeted task is not classification but segmentation

Reply:
Concernant cette phrase, on voit un class token sur ta première figure, non ?

Reply:
Oui, je me suis fait la même remarque, mais je pense que c'est juste une erreur sur la figure